src/main/java/com/targetindia/DigitizeTimeTable/repository/ConnectionDao.java

```diff
@@ -0,0 +1,31 @@
+package com.targetindia.DigitizeTimeTable.repository;
+
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+
+public class ConnectionDao {
+
+
+    Connection conn = null;
+
+    String url="jdbc:postgresql://localhost:5433/DigitizeTimeTable";
+    String username="postgres";
+    String password="postgres@123";
```

Comment:
move this file to configs not here. also externalise the passwords from application.yaml